portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>My Portfolio</title>
+    
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" 
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" 
+        crossorigin="anonymous">   
+
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" 
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" 
+        crossorigin="anonymous">
+    </script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" 
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" 
+        crossorigin="anonymous">
+    </script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" 
+        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" 
+        crossorigin="anonymous">
+    </script>
+
+    <!-- Regular CSS -->
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <img src="images/profile.jpg" width="150"
+        class="rounded-circle mx-auto mt-3 d-block img-fluid p-0 border border-light">
+    <nav class="navbar navbar-expand-sm justify-content-center">
+      <ul class="navbar-nav">
+        <li class="nav-item ">
+          <a class="nav-link text-light" href="index.html">PROJECTS</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-light" href="about.html">ABOUT</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-light" href="contact.html">CONTACT</a>
+        </li>
+      </ul> 
+    </nav>
+
+    <div class="d-flex justify-content-center mt-5">
```

Comment:
Side note: this whole file is really clean HTML. I don't think I could write code this well-formatted if my life depended on it.